.clang-format

```diff
@@ -72,6 +72,8 @@ IncludeCategories:
     Priority: 110
   - Regex:    "deal.II/base/.*\\.h"
     Priority: 120
+  - Regex:    "deal.II/boost_adaptors/.*\\.h"
+    Priority: 125
```

Comment:
I am glad that I left all of the empty space in the integer range. This is exactly the right place for a new directory :)